updater/lib/dependabot/api_client.rb

```diff
@@ -18,10 +20,11 @@
 module Dependabot
   class ApiError < StandardError; end
 
-  class ApiClient
+  class ApiClient # rubocop:disable Metrics/ClassLength
```

Comment:
nit, we could satisfy the cop if we extracted some logic out into it's own method

Reply:
It's the class length so we would have to  extract a mixin but then extracting a mixin just for this would probably not be worth it.